src/docs/teamFMS/chromeDev.md

```diff
@@ -0,0 +1,43 @@
+# Developing with local chrome
+
+## Prerequisites
+
+1. Have insights-chrome cloned
+2. Have insights-proxy cloned
+    * Make sure to run the update.sh script and have your hosts set up
+3.Have an application cloned
+
+## Building Chrome
+
+To build chrome, you can run `npm run build` to build the entire project once or `npm run watch` to watch changes across all files and rebuild after every save
+Proxy Flag
+
+The `LOCAL_CHROME=true` flag lets developers set insights-proxy to use a local version of insights-chrome instead of the version that is on ci-beta
+
+Your full command should look something like:
+
+```sh
+SPANDX_CONFIG=../../insights-frontend-starter-app/profiles/local-frontend.js sh ../../insights-proxy/scripts/run.sh
+```
+
+Note: Different apps use different profiles. Because of this, your spandx_config file might be different than local-frontend.js
```

Comment:
Should probably have a period at the end, but maybe that would look weird right after a filename?

Reply:
Wrapping `local-frontend.js` in code tags and then adding period.